library/cirros

```diff
@@ -3,11 +3,20 @@
 Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon)
 GitRepo: https://github.com/tianon/docker-brew-cirros.git
 GitFetch: refs/heads/dist
-GitCommit: 84bfc9d0e4cc631d043410d602bf5f8876ed6575
+GitCommit: e8833253f108046f977fbcecd7f02170bd20d357
+Architectures: amd64, arm32v7, arm64v8, ppc64le
+amd64-Directory: arches/amd64
+arm32v7-Directory: arches/arm32v7
+arm64v8-Directory: arches/arm64v8
+ppc64le-Directory: arches/ppc64le
+
+Tags: 0.6.2, 0.6, 0, latest
+
+Tags: 0.5.3, 0.5
```

Comment:
Reviewer's note: this is in kind of a weird format for now because I expect this 0.5.x entry to be temporary (I don't know whether CirrOS 0.5.x is going to be actively supported still :sweat_smile:)